src/ApiService/ApiService/onefuzzlib/Reports.cs

```diff
@@ -28,9 +28,9 @@ public class Reports : IReports {
 
     public async Async.Task<IReport?> GetReportOrRegression(Container container, string fileName, bool expectReports = false, params string[] args) {
         var filePath = string.Join("/", new[] { container.String, fileName });
-        if (!fileName.EndsWith(".json", StringComparison.Ordinal)) {
+        if (!fileName.EndsWith(".json", StringComparison.Ordinal) || fileName.Contains("source-coverage", StringComparison.InvariantCultureIgnoreCase)) {
```

Comment:
Should this not be:
```suggestion
        if (!fileName.EndsWith(".json", StringComparison.Ordinal) || !fileName.Contains("source-coverage", StringComparison.InvariantCultureIgnoreCase)) {
```

Reply:
@tevoinea can confirm, but it was my understanding that we don't want to parse the file if its name starts `source-config`.

Reply:
That's right, we don't want to parse the file is the name _ends with_ `source-coverage.json`